Backpack/src/internal/res/values/backpack.font.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <style name="BackpackFont" parent="BackpackFontBase">
+    <item name="bpkFontFamilyBase">monospace</item>
```

Comment:
Can we keep the normal sans serif font here for now? Monospace was just a test but we don't want people using this by mistake.